.github/workflows/system-tests.yml

```diff
@@ -4,9 +4,11 @@ on:
   push:
     branches:
       - "**"
+    paths-ignore:
+      - "docs/**"
```

Comment:
This is a nice idea! TBH I think a lot more things could be added there -- `.circleci`/`.gitlab`/`appraisal`/`benchmarks`/`gemfiles`/`integration`/`sig`/`spec`/`suppressions`/`tools`/`vendor`.

None of these have any impact on released code, and thus would not impact system-tests.

Up to you if you want to do that as well -- I understand this would increase the scope of this PR a lot and you may not wanna ;)